_opensearch/rest-api/document-apis/update-document.md

```diff
@@ -54,7 +54,7 @@ require_alias | Boolean | Specifies whether the destination must be an index ali
 refresh | Enum | If true, OpenSearch refreshes shards to make the operation visible to searching. Valid options are `true`, `false`, and `wait_for`, which tells OpenSearch to wait for a refresh before executing the operation. Default is `false`. | No
 retry_on_conflict | Integer | The amount of times OpenSearch should retry the operation if there's a document conflict. Default is 0. | No
 routing | String | Value to route the update operation to a specific shard. | No
-_source | List | Whether to include the `_source` field in the response body. Default is true. | No
+_source | Boolean or List | Whether to include the `_source` field in the response body. Default is `false`. This parameter can also be used with a comma-separated list of source fields to include in the query response. | No
```

Comment:
Just suggestions.
"Specifies whether to populate the `_source` field in the response body."? 
Maybe, "This parameter also supports a comma-separated list of source fields for including multiple source fields in the query response."